### PR TITLE
community page content updates

### PR DESCRIPTION
**Pull request by michow**
Signed-off-by: michow <michow@smalltalk.agency>

## Which issue(s) this PR fixes

Fixes: #

### CODE REVIEW

**Comment by dvonthenen** [comment]
This looks ok to me, but want to get @kcoriordan feedback on it

**Review by davidvonthenen — DISMISSED** [review]

**Review by stmcginnis — DISMISSED** [review]
Looks good to me.

**Comment by microwavables** [comment]
Sorry - need to change the link for agenda / notes for monthly community meetings after a meeting I had today...

File: docs/site/content/community/_index.html [hunk 1]
```diff
@@ -39,14 +40,21 @@ <h3><a href="https://kubernetes.slack.com/archives/C02GY94A8KT">Join the convers
...
-                    <li>See previous community meetings on our <a href="https://www.youtube.com/playlist?list=PL7bmigfV0EqTjEFkgebV1RKoAGax2N_sY">YouTube Playlist</a></li>
-                    <li><a href="https://github.com/vmware-tanzu/community-edition/discussions">Meeting notes</a></li>
+                    <li>Wednesdays at 11:00 AM PT/ 2:00 PM ET/ 8:00 PM CET <em>(<a href="https://dateful.com/time-zone-converter?t=11:00&tz=PT%20%28Pacific%20Time%29" target="_blank">convert to your time zone</a>)</em></li>
+                    <li><a href="https://hackmd.io/CiuO4V0AT6WL_TgA47MXBA">Meeting Agenda and Notes</a></li>
+                    <li><a href="https://vmware.zoom.us/j/91225878366?pwd=R3dIMXVxRDZ0Qnk1RXdkcmNKSitidz09">Zoom Link</a></li>
                 </ul>
+                <p>Monthly meetings:</p>
+                <ul>
+                    <li>Second Thursday of each month at 9:00 AM IST <em>(<a href="https://dateful.com/time-zone-converter?t=9:00&tz=IST" target="_blank">convert to your time zone</a>)</em></li>
+                    <li><a href="https://hackmd.io/CiuO4V0AT6WL_TgA47MXBA">Meeting Agenda and Notes</a></li>
```
Comment (microwavables) [inline docs/site/content/community/_index.html:54]:
```suggestion
                    <li><a href="https://hackmd.io/EA962cUBRwODRluxTtW_Yw">Meeting Agenda and Notes</a></li>
```

**Review by microwavables — CHANGES_REQUESTED** [review]

**Review by kcoriordan — DISMISSED** [review]

**Review by stmcginnis — APPROVED** [review]

**Pull request merged**